lib/faker/default/id_number.rb

```diff
@@ -94,6 +160,17 @@ def brazilian_citizen_number(legacy_formatted = NOT_GIVEN, formatted: false)
 
       alias brazilian_cpf brazilian_citizen_number
 
+      ##
+      # Produces a random Brazilian ID Number (RG).
+      #
+      # @param formatted [Boolean] Specifies if the number is formatted with dividers.
+      # @return [String]
+      #
+      # @example
+      #   Faker::IDNumber.brazilian_id #=> 1212312312
+      #   Faker::IDNumber.brazilian_id(formatted: true) #=> 12.123.123-12
```

Comment:
These are invalid RGs. Maybe we could use a valid RG number?
Also, the format seems to be incorrect
```suggestion
      #   Faker::IDNumber.brazilian_id #=> "493054029"
      #   Faker::IDNumber.brazilian_id(formatted: true) #=> "49.305.402-9"
```

Reply:
Too much reliance on the docs. Thank you for catching it!